pkg/util/ipvs/ipvs.go

```diff
@@ -129,9 +131,9 @@ func GetRequiredIPVSModules(kernelVersion *version.Version) []string {
 	// "nf_conntrack_ipv4" has been removed since v4.19
 	// see https://github.com/torvalds/linux/commit/a0ae2562c6c4b2721d9fddba63b7286c13517d9f
 	if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
-		return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrackIPV4}
+		return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrackIPV4, KernelModuleIPVSSED}
 	}
-	return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrack}
+	return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrack, KernelModuleIPVSSED}
```

Comment:
In the past we haven't been checking for all the IPVS scheduling modules aside from the really common ones and the one we use by default (ip_vs_rr).  I don't think it's worthwhile to stop kube-proxy if this module is missing but we should definitely document that it should be loaded if it's the desired scheduling algorithm.

Reply:
Alternatively we can check the existence of the module based on the passed in scheduling algorithm as opposed to checking a set of scheduling algorithms every time.

Reply:
Ok I think the alternate check for checking for the existence if that algorithm is selected is a better fix. I will go with that.